app/helpers/data_table_helper.rb

```diff
@@ -0,0 +1,30 @@
+module DataTableHelper
+  REPORT_LABELS = {
+    :striped_hover => "table table-striped table-bordered table-hover",
+    :striped       => "table table-striped table-bordered",
+    :carbon        => "bx--data-table bx--data-table--normal bx--data-table--no-border miq-data-table",
+    :clickable     => "clickable",
+    :blank         => "No records found"
```

Comment:
Actually this is going to be tough because you're using the blank string in a String#include? check, which probably won't play nice with translation.  Might be better to have something like a CSS class or something for blank records.